modules/fastani/fastani.nf

```diff
@@ -1,40 +0,0 @@
-// TODO
-
-nextflow.enable.dsl = 2
-
-params.resultsDir = "${params.outdir}/unicycler"
```

Comment:
Have you removed these modules?

I do intend to complete the `bwa` and `fastani` in future over here.

Reply:
yes, sorry! I'll restore them now.